src/main/java/com/vecna/maven/db/mysql/MysqlStrategy.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2014 Vecna Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.vecna.maven.db.mysql;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.List;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.codehaus.plexus.util.cli.CommandLineException;
+import org.codehaus.plexus.util.cli.CommandLineUtils;
+import org.codehaus.plexus.util.cli.CommandLineUtils.StringStreamConsumer;
+import org.codehaus.plexus.util.cli.Commandline;
+
+import com.google.common.collect.Lists;
+import com.vecna.maven.db.DbStrategy;
+
+/**
+ * MySQL strategy for creating/dropping/dumping databases.
+ * @author jyoung@vecna.com
+ */
+public class MysqlStrategy implements DbStrategy {
+  private static final String DEFAULT_DB = "/mysql";
```

Comment:
Hmm, the preceding slash in "/mysql" seems a little odd to me. I think my inclination would be to add it later near line 86 (which would match the test that calls the db "db1").